pvlib/pvsystem.py

```diff
@@ -1098,20 +1104,15 @@ class Array:
     """
     An Array is a set of of modules at the same orientation.
 
-    Specifically, an array is defined by tilt, azimuth, the
+    Specifically, an array is defined by its mount, the
     module parameters, the number of parallel strings of modules
     and the number of modules on each string.
 
     Parameters
     ----------
-    surface_tilt: float or array-like, default 0
-        Surface tilt angles in decimal degrees.
-        The tilt angle is defined as degrees from horizontal
-        (e.g. surface facing up = 0, surface facing horizon = 90)
-
-    surface_azimuth: float or array-like, default 180
-        Azimuth angle of the module surface.
-        North=0, East=90, South=180, West=270.
+    mount: FixedMount, SingleAxisTrackerMount, or other, default None
+        Mounting strategy for the array, used to determine module orientation.
+        If not provided, a FixedMount with zero tilt is used.
```

Comment:
Perhaps make this a required parameter and let `PVSystem` handle the default case for backwards compatibility.

"or other" has me thinking we might want to define an abstract base class for mounts.

Reply:
I agree with an ABC for mounts. I think it goes a long way in terms of documentation, if nothing else.